src/AspNetCoreHealth/sandbox/HealthSandboxMvc/HealthSandboxMvc.csproj

```diff
@@ -24,8 +24,8 @@
     <PackageReference Include="App.Metrics.Health.Checks.Process" />
     <PackageReference Include="App.Metrics.Health.Reporting.Metrics" />
     <PackageReference Include="App.Metrics.Health.Reporting.Slack" />
-    <PackageReference Include="Microsoft.AspNetCore" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc" />
+    <PackageReference Include="Microsoft.AspNetCore" Version="2.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="2.2.0" />
```

Comment:
I think this one above is also target for this PR. It's referenced in more than one place.

Reply:
I left the Sandboxes targetting 2.2 by intention. As I undestood this is playground for new features, so it doesnt really matter.

Reply:
ok :)